tests/DefaultConfigValuesTest.php

```diff
@@ -30,7 +30,7 @@ public function setUp(): void
     /** @test */
     public function secretShouldBeNull()
     {
-        $this->assertNull($this->configuration['secret']);
+        $this->assertEquals($this->configuration['secret'], '');
```

Comment:
I suggest changing the method name to `secretShouldBeEmpty`

Reply:
You are totally right. I have changed the name.

Reply:
@avidianity changes were made.